include/fmt/printf.h

```diff
@@ -257,7 +257,7 @@ class printf_arg_formatter : public detail::arg_formatter_base<Range> {
         return (*this)(static_cast<int>(value));
       fmt_specs.sign = sign::none;
       fmt_specs.alt = false;
-      fmt_specs.align = align::right;
+      if (fmt_specs.align == align::numeric) fmt_specs.align = align::right;
```

Comment:
I'm not sure which possible values of align really have to be overwritten here. align::numeric is the one case where a test fails if it isn't overwritten.

Reply:
I think this should be `align::none`, not `align::numeric`, because none is the default and here we are trying to change the default to right alignment.